python/lsst/ts/eas/dome_model.py

```diff
@@ -83,12 +94,24 @@ def is_closed(self) -> bool | None:
         if self.aperture_shutter_telemetry is None:
             return None
 
-        send_timestamp = self.aperture_shutter_telemetry.private_sndStamp
+        if self.louvers_telemetry is None:
+            return None
+
+        send_timestamp = min(
+            self.aperture_shutter_telemetry.private_sndStamp,
+            self.louvers_telemetry.private_sndStamp,
+        )
         telemetry_age = utils.current_tai() - send_timestamp
         if telemetry_age > MAX_TELEMETRY_AGE:
             return None
 
-        return (
+        shutters_closed = (
             self.aperture_shutter_telemetry.positionActual[0] < DOME_OPEN_THRESHOLD
             and self.aperture_shutter_telemetry.positionActual[1] < DOME_OPEN_THRESHOLD
         )
+        louvers_closed = all(
+            position < DOME_OPEN_THRESHOLD
```

Comment:
Can the DOME_OPEN_THRESHOLD become a configurable variable - louvers_open_threshold? We recently had a case when the summit opened louvers at 30 %, better we made this a config variable.